CHANGELOG.rst

```diff
@@ -0,0 +1,109 @@
+2018.05.18 (2018-06-17)
+=======================
```

Comment:
This makes no sense 😂

Reply:
shhhhh

Reply:
and dont look at any of the followup commits that got buildkite working :p